src/utils/checkError.js

```diff
@@ -0,0 +1,41 @@
+import {
+  IS_SELECTOR_WRONG,
+  IS_WRONG_NUMBERS,
+  IS_WRONG_SEPERATOR,
+} from "../data/constants.js";
+import {
+  WRONG_INPUT_ERROR,
+  WRONG_NUMBERS_ERROR,
+  WRONG_SELECTOR_ERROR,
+  WRONG_SEPERATOR_ERROR,
+} from "../data/errorMessage.js";
+
+// string => Error?
+// 입력받은 값 유효성 검사
+const checkError = (restString, seperatorJoinString) => {
+  const numbersRegexString = `^\\d+((${seperatorJoinString})\\d+)*$`;
+  const numbersRegex = new RegExp(numbersRegexString);
+
+  // 에러 없음
+  if (numbersRegex.test(restString) | (restString === "")) {
+    return;
+  }
```

Comment:
```suggestion
  if (numbersRegex.test(restString) | (restString === "")) return;
```
아무 것도 반환하지 않는다면 이렇게 작성하는 것은 어떻게 생각하세요? 한 줄 코딩?

Reply:
저야 너무 좋은데 혹시 컨벤션에 저촉되지는 않나요..?